### DISCUSSION TITLE
Get the content of a message with its message ID

### DISCUSSION THREAD

**Discussion by opipoy** [rigwild/discord-self-bot-console#65] [Q&A] [answered]
is there a way to get a specific message using the api and not loop through the getMessages until i get to my specific message. i saw there is a thing called "Message Id" and that you use it to delete a message. i was wondering if i could use that for getting a specific message or just as mentioned above loop until i get to my desired message?

**Comment by rigwild** [comment, marked as answer]
Hey, there is an endpoint for this [`GET /channels/{channel.id}/messages/{message.id}`](https://discord.com/developers/docs/resources/channel#get-channel-message)

I tried to call it and get the following error:

```json
{
    "message": "Only bots can use this endpoint",
    "code": 20002
}
```

They block access for normal users, which kinda make sense as there is no real use case for normal users outside of automated behavior, that they disallow
So unfortunately, it is not possible

**Reply by opipoy** [reply]
Ok so just looping through the messages. Thank you for the fast responce
